drivers/power_domain/power_domain_intel_adsp.c

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <kernel.h>
+#include <pm/device.h>
+#include <pm/device_runtime.h>
```

Comment:
I think due to a recent change, these need to have "zephyr/" prefix.

Reply:
Agree, chenaged